chsdi/templates/htmlpopup/tww_anhang2.mako

```diff
@@ -1,17 +1,33 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c, lang)">
-    <tr><td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.objnummer')}</td>          <td>${c['attributes']['objnummer'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.name')}</td>         <td>${c['attributes']['name'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.teilobjnummer')}</td>          <td>${c['attributes']['teilobjnummer'] or '-'}</td></tr>
-    <tr>
-      <td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.shape_area')}</td>
-      % if c['attributes']['shape_area']:
-         <td>${round(c['attributes']['shape_area'] / 10000, 1)}</td>
-      % else:
-         <td>-</td>
-      % endif
-    </tr>
-    <tr><td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.refobjblat')}</td>        <td><a target="_blank" href="${c['attributes']['refobjblat']}">${_('link') or '-'}</a></td></tr>
+   <tr>
+      <td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.objnummer')}</td>
+      <td>${c['attributes']['objnummer'] or '-'}</td>
+   </tr>
+   <tr>
+      <td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.name')}</td>
+      <td>${c['attributes']['name'] or '-'}</td>
+   </tr>
+   <tr>
+      <td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.teilobjnummer')}</td>
+      <td>${c['attributes']['teilobjnummer'] or '-'}</td>
+   </tr>
+   <tr>
+      <td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.singularit')}</td>
+      <td>${c['attributes']['singularit'] or '-'}</td>
+   </tr>
+   <tr>
+     <td class="cell-left">${_('ch.bafu.bundesinventare-trockenwiesen_trockenweiden_anhang2.shape_area')}</td>
+     % if c['attributes']['shape_area']:
+        <td>${round(c['attributes']['shape_area'] / 10000, 1)}</td>
```

Comment:
I am not sure we should manipulate the data here in the tool-tip. But since it has been like this for a long time I guess it is OK